examples/testbed_native_android/android/app/src/main/AndroidManifest.xml

```diff
@@ -26,6 +26,12 @@
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
             </intent-filter>
+            <intent-filter>
```

Comment:
Please change to `<intent-filter android:autoVerify="true"> `  to support automatic app link verification through DAL